mne/utils/docs.py

```diff
@@ -271,11 +271,12 @@ def _reflow_param_docstring(docstring, has_first_line=True, width=75):
 # raw/epochs/evoked apply_function method
 # apply_function method summary
 applyfun_summary = """\
-The function ``fun`` is applied to the channels or vertices defined in ``picks``.
-The {} object's data is modified in-place. If the function returns a different
+The function ``fun`` is applied to the {applies_to} defined in ``picks``.
+The {data_type} object's data is modified in-place. If the function returns a different
 data type (e.g. :py:obj:`numpy.complex128`) it must be specified
 using the ``dtype`` parameter, which causes the data type of **all** the data
-to change (even if the function is only applied to channels/vertices in ``picks``).{}
+to change (even if the function is only applied to {applies_to} in ``picks``).
+{preload}
```

Comment:
Otherwise you end up with two linebreaks in the empty-preload case which is a style error
```suggestion
to change (even if the function is only applied to {applies_to} in
``picks``). {preload}
```